tests/Deedle.Tests/Series.fs

```diff
@@ -655,10 +655,25 @@ let b =
   [ DateTime(2013,9,8) => 8.0; // no matching point in a
     DateTime(2013,9,11) => 11.0 ] |> series
 
+let dense =
+  [ 1 => 1.0; 
+    2 => 2.0; 
+    3 => 3.0;
+    4 => 4.0;
+    5 => 5.0; ]
+    |> series
+
+let sparse = 
+  [ 2 => 20.0;
+    4 => 40.0 ] |> series
+
 let lift2 f a b = 
     match a, b with
     | Some x, Some y -> Some(f x y)
-    | _              -> None
+    | None, Some y              -> None
+    | Some x, None              -> None
+    | None, None                -> None
+    | _                 -> failwith "expecting an option"
```

Comment:
Hmm.. perhaps that was coming from somewhere else? Anyway, would you mind removing this part of the change? (I think this might even give you a "case will never be matched" compiler warning...)